.github/workflows/ci-slow.yml

```diff
@@ -142,11 +142,12 @@ jobs:
       - name: Create virtual environment
         run: |
           uv venv
-      - name: Check mlstacks compatibility
-        run: |
-          source .venv/bin/activate
-          uv pip install -e .
-          uv pip install mlstacks
+#      TODO: REACTIVATE BEFORE THE RELEASE
```

Comment:
TODO

Reply:
I am aware of this of these TODOs. I added them to the list in the description. For the sake of clarity, resolving these two.

Reply:
@bcdurak seems like this one got merged anyways? Was that intended?